pylint/checkers/variables.py

```diff
@@ -170,7 +170,7 @@
 
 
 class VariableVisitConsumerAction(Enum):
-    """Used after _visit_consumer to determine the action to be taken
+    """Used after _check_consumer to determine the action to be taken
```

Comment:
This is out of date right? Now that `_is_first_level_self_reference` uses it as well? Perhaps we should make the docstring more general and not refer to method names that could change.

Reply:
Hmm, on further thought. It is still used in `check_consumer` but also in "sub-methods". I would still think a more general docstring makes more sense here.